raiden-cli/raspberrypi.md

```diff
@@ -0,0 +1,92 @@
+# Running raiden-cli on the raspberry pi
+
+This tutorial assumes raspberry pi to be in the same LAN as your PC with SSH enabled
+These steps have to be run on the raspberry pi 4. The steps to copy files from your `PC` to the `pi` have been explicitly mentioned and have to be run on your `PC`.
+
+`sudo apt update`
+
+`sudo apt full-upgrade`
+
+`sudo apt install git`
+
+Perform a [git install](https://github.com/nvm-sh/nvm#git-install) of the NVM
+
+`git clone https://github.com/nvm-sh/nvm.git .nvm`
+
+`cd ~/.nvm`
+
+`git checkout v0.38.0`
+
+`. ./nvm.sh`
+
+`nvm install --lts`
+
+`npm install --global yarn`
+
+Check the arm system
```

Comment:
for what should I look out?

Reply:
There are many systems and depending on the OS your system could show you a different `arm` system. Mainly I did it for geth because it supports many different arm systems